Doc/source/index.rst

```diff
@@ -39,15 +51,23 @@ To cite SpacePy in publications, use (BibTeX code):
     address = {Austin, TX}
     }
 
-Or to cite the code itself:
-    @software{SpacePy,
-    author       = {{Larsen}, B.~A. and {Morley}, S.~K. and {Niehof}, J.~T. and {Welling}, D.~T.},
+To cite the code itself:
+    @software{spacepy_code,
+    author       = {Morley, Steven K. and Niehof, Jonathan T. and
+                    Welling, Daniel T. and Larsen, Brian A. and
+                    Brunet, Antoine and Engel, Miles A. and
+                    Gieseler, Jan and Haiducek, John and
+                    Henderson, Michael and Hendry, Aaron and
+                    Hirsch, Michael and Killick, Peter and
+                    Koller, Josef and Merrill, Asher and
+                    Rastatter, Lutz and Reimer, Ashton and
+                    Shih, Albert Y. and Stricklan, Amanda},
     title        = {SpacePy},
     publisher    = {Zenodo},
     doi          = {10.5281/zenodo.3252523},
     url          = {https://doi.org/10.5281/zenodo.3252523}
     }
-
+                                                                                                                                                                                                                                                                                                                                                                                                                                    }
```

Comment:
I think this should go away entirely....the brace is already closed and this is a bunch of whitespace before a "bonus" brace.

Reply:
This is probably the reason for the docs CI failing

Reply:
I deleted that, amended the commit and force pushed. Clearly one step did not succeed and I failed to notice. I'll get to it on Monday.

Reply:
This PR is against the central repo not your fork, so maybe that's part of what happened?